analysis/src/Diagnostics.ml

```diff
@@ -0,0 +1,26 @@
+let document_syntax ~path =
+  let parse =
+    Res_driver.parsingEngine.parseImplementation ~forPrinter:false
```

Comment:
See other commands on how to parse implementation/interface

so `.res` and `.resi` files are supported.